cdap-docs/reference-manual/source/faq.rst

```diff
@@ -96,7 +96,7 @@ Yes. You can install CDAP on your Hadoop cluster. See :ref:`install`.
 
 .. rubric:: What Hadoop distributions can CDAP run on?
 
-CDAP has been tested on and supports CDH 4.2.x or later, HDP 2.0 or later, and Apache Hadoop/HBase 2.0.2-0.4 and 2.1.0. 
+CDAP has been tested on and supports CDH 4.2.x through 5.3.x, HDP 2.0 through 2.1, and Apache Hadoop/HBase 2.0.2, 2.0.4, and 2.1.0. 
```

Comment:
I'm wondering if even that's correct: should it not be (too match the chart above):
```
 ...and Apache Hadoop 2.0.2-alpha through 2.5.0
```
otherwise, it would seem odd to me to see once sequence in one place and a different one elsewhere.